docs/guides/admin/docs/modules/index.md

```diff
@@ -15,7 +15,7 @@ Documentation for modules included in Opencast.
     * [URL Parameter](player.url.parameter.md)
 * [Search Indexes](searchindex/index.md)
     * [Solr](searchindex/solr.md)
-    * [Elasticsearch](searchindex/elasticsearch.md)
+    * [Elasticsearch](../configuration/elasticsearch.md)
```

Comment:
It shouldn't be listed in the modules section if it's part of the configuration section. But then… these sections do not make a lot of sense anyway…

Reply:
I'll remove it, you're right.